File: app/src/main/java/com/miaadrajabi/menuxmlparser/java/MainActivity.java
package com.miaadrajabi.menuxmlparser.java;

import android.content.Intent;
import android.os.Bundle;

import androidx.activity.ComponentActivity;

import com.miaadrajabi.menuxmlparser.java.presentation.menu.MenuActivity;

public class MainActivity extends ComponentActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        startActivity(new Intent(this, MenuActivity.class));
        finish();
    }
}

File: app/src/main/java/com/miaadrajabi/menuxmlparser/MainActivity.kt
package com.miaadrajabi.menuxmlparser

import android.content.Intent
import android.os.Bundle
import androidx.activity.ComponentActivity
import com.miaadrajabi.menuxmlparser.presentation.menu.MenuActivity

class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        startActivity(Intent(this, MenuActivity::class.java))
        finish()
    }
}